benches/batch.rs

```diff
@@ -89,8 +87,6 @@ fn partitioned_batching(
             },
         )
     })
-    .sample_size(10)
-    .noise_threshold(0.05)
     .throughput(Throughput::Bytes((num_events * event_len) as u64))
 }
 
```

Comment:
Below, in this file, theyre are a bench of benches calling this code that use high `num_events` numbers, and I repeat myself about coarse grained samples and criterion already doing this.